.github/workflows/cd.yml

```diff
@@ -0,0 +1,30 @@
+name: CD
+
+# Run on Pull Requests to master and on manual interaction
+on:
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  deploy-to-dev:
+    runs-on: ubuntu-latest
+    concurrency:
+      group: histalek-dev-env
+      cancel-in-progress: true
+    environment:
+      name: histalek-dev-env
+      #url: https://example.com
```

Comment:
This could be set to something along the lines of `steam://connect/<domain>:<port>/<password>` to ease connecting to the server (IF it works properly).

Obviously that would allow anyone to join the server. If that's unwanted we could easily omit the password part.

Also regarding that steam:// link: AFAIK that depends on Steam's `Master Server Query Protocol` [1] so the server would have to be started **without** specifying `-nomaster` in the start command. 

[1] https://developer.valvesoftware.com/wiki/Master_Server_Query_Protocol